package.json

```diff
@@ -34,7 +34,7 @@
   },
   "sideEffects": false,
   "devDependencies": {
-    "@checkdigit/jest-config": "^6.0.0",
+    "@checkdigit/jest-config": "6.0.0",
```

Comment:
why fixed to 6.0.0?

Reply:
It fails saying process.loadEnvFile and we don't have any env file.

Reply:
ok I created an issue for this, https://github.com/checkdigit/jest-config/issues/33